package.json

```diff
@@ -56,6 +56,7 @@
         "@commitlint/config-conventional": "^8.3.4",
         "@nativescript/angular":"8.21.0",
         "@nativescript/core":"6.5.1",
+        "@types/node": "14.0.4",
```

Comment:
There's an issue with the latest node typings (14.0.3) where ngc isn't working, so I just forced it to use 14.0.4. I can send out another PR to remove this once the node typings work again.